op-e2e/system_test.go

```diff
@@ -364,6 +364,35 @@ func TestFinalize(t *testing.T) {
 	require.NotZerof(t, l2Finalized.NumberU64(), "must have finalized L2 block")
 }
 
+// TestDencunFinalize tests if L2 finalizes in a post-4844 environment
+func TestDencunFinalize(t *testing.T) {
+	InitParallel(t)
+
+	cfg := DefaultSystemConfig(t)
+	//FIXME how do we turn on dencun/cancun? need to edit the geth genesis but where is that?
```

Comment:
Turning on Dencun is handled in the deploy-config. I previously implemented support for that in the 4844 branch, and pulled it into https://github.com/ethereum-optimism/optimism/pull/8131